### PR TITLE
fetched movies

### PR DESCRIPTION
**Pull request by silchyn**
[DEMO](https://silchyn.github.io/react_movies-list-fetch-movies/)

### CODE REVIEW

File: src/components/FindMovie/FindMovie.tsx [hunk 1]
```diff
@@ -17,20 +32,60 @@ export const FindMovie: React.FC = () => {
...
+                    imdbUrl: `https://www.imdb.com/title/${response.imdbID}`,
+                    imdbId: response.imdbID,
+                  });
+                } finally {
+                  setLoading(false);
+                }
+              }}
+              disabled={!canFind || !title}
             >
               Find a movie
```
Comment (rodiontseva) [inline src/components/FindMovie/FindMovie.tsx:90]: need to change to ''Search again'', if movie wasn't add to list (https://prnt.sc/Axr2gPusoYOM)

File: src/components/FindMovie/FindMovie.tsx [hunk 2]
```diff
@@ -1,7 +1,22 @@
...
+  movies: Movie[],
+  setMovies(movies: Movie[]): void,
+};
+
+export const FindMovie: React.FC<Props> = ({ movies, setMovies }) => {
+  const [title, setTitle] = useState('');
+  const [showError, setShowError] = useState(false);
+  const [currentMovie, setCurrentMovie] = useState<Movie | null>(null);
+  const [canFind, setCanFind] = useState(false);
+  const [loading, setLoading] = useState(false);
```
Comment (rodiontseva) [inline src/components/FindMovie/FindMovie.tsx:18]: better isLoading (if we fave boolean)

**Review by rodiontseva — CHANGES_REQUESTED** [review]
good job! some comments

**Review by rodiontseva — APPROVED** [review]
👌

**Review by ivan-kadykalo — APPROVED** [review]
🔥

**Pull request closed without merging**